langtest/langtest.py

```diff
@@ -1038,11 +1072,11 @@ def edit_testcases(self, output_path: str, **kwargs):
         temp_df = temp_df[temp_df["category"].isin(["robustness", "bias"])]
         temp_df.to_csv(output_path, index=False)
 
-    def import_edited_testcases(self, input_path: str, **kwargs):
+    def import_edited_testcases(self, input_path: dict, **kwargs):
```

Comment:
user can provide only the **str of the input path**. and we can convert it into the **required dict format** within the function.